pages.es/common/vim.md

```diff
@@ -0,0 +1,37 @@
+# vim
+
+> Vim (Vi IMproved), un editor de texto para la línea de comandos, que proporciona varios modos para diferentes tipos de manipulación de texto.
+> Pulsando `i` entra en el modo insertar. `<Esc>` habilita el uso de comandos de Vim.
```

Comment:
We're currently rephrasing this line in #6237, but we didn't agree on that yet, so it would be nice if we get that one merged first and then apply it here too.